### PR TITLE
recipe for bootstrap_contrast

### PR DESCRIPTION
**Pull request by josesho**
http://github.com/josesho/bootstrap_contrast

### CODE REVIEW

File: recipes/bootstrap_contrast/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/bootstrap_contrast/meta.yaml]: Please delete all comments for the final version of the recipe.

File: recipes/bootstrap_contrast/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
```
Comment (synapticarbors) [inline recipes/bootstrap_contrast/meta.yaml]: It looks like you need to add `setuptools` to the `build` requirements list.

File: recipes/bootstrap_contrast/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  run:
+    - python
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - matplotlib
```
Comment (synapticarbors) [inline recipes/bootstrap_contrast/meta.yaml]: You shouldn't include all of these packages under the `test`/`imports` section. You only really need `bootstrap_contrast` here.

File: recipes/bootstrap_contrast/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+  run:
+    - python
```
Comment (synapticarbors) [inline recipes/bootstrap_contrast/meta.yaml]: You should add all of the packages that are run time requirements of `bootstrap_contrast` here. Probably everything listed:

https://github.com/josesho/bootstrap_contrast/blob/master/setup.py#L16-L42

including the proper version restrictions.

File: recipes/bootstrap_contrast/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: GPL-3.0
+  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
+  license_family: GPL
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See http://conda.pydata.org/docs/building/meta-yaml.html#license-file
+  # license_file: LICENSE.txt
```
Comment (synapticarbors) [inline recipes/bootstrap_contrast/meta.yaml]: You should include the `license_file`, either by cutting a new upstream release or by temporarily adding the license to the recipe as described here:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

**Comment by josesho** [comment]
OK I will pull a fresh set of changes in the morning. Thanks!

**Pull request closed without merging**